### ISSUE TITLE
Case mismatch? org.gtk.Gtk3theme.Yaru-Red-dark vs Yaru-red-dark

### ISSUE THREAD

**Issue by ernstp** [flathub/org.gtk.Gtk3theme.Yaru-Red-dark#1]
I get legacy integration with Evolution from flathub when I have "Yaru-dark" selected, but not with "Yaru-red-dark".
Is it because the flatpak theme is called Yaru-Red-dark with an uppercase R perhaps.. ?

**Comment by Mek101** [comment]
That should the cause. I don't know for sure which capitalization is right though. From where did you get get it from?

**Comment by ernstp** [comment]
In gnome-tweaks you can see that the theme name is "Yaru-red-dark". And this Flatpak stuff is named Yaru-Red-dark.

**Comment by Mek101** [comment]
How are the other colors capitalized?

**Comment by ernstp** [comment]
It's the same pattern with all the other colors!

**Comment by 3v1n0** [comment]
The problem is not only on naming, but mostly that this repo should now be rebased on upstream Yaru that supports color variants.

Old variants are not supported upstream, where just a fork.

**Comment by Mek101** [comment]
> The problem is not only on naming, but mostly that this repo should now be rebased on upstream Yaru that supports color variants.

I just package the themes into flatpaks, you should go and ask at  https://github.com/Jannomag/Yaru-Colors for an upstream rebase

> Old variants are not supported upstream, where just a fork.

I... don't understand this phrase

**Comment by 3v1n0** [comment]
> I just package the themes into flatpaks, you should go and ask at https://github.com/Jannomag/Yaru-Colors for an upstream rebase

Nope, I don't expect that repository to be maintained as it is, now upstream [Yaru](https://github.com/ubuntu/yaru/) supports different colors natively so this repo should be changed to track that instead. 

>>  Old variants are not supported upstream, where just a fork.

> I... don't understand this phrase

Sorry I put an extra `h`, but I meant that Yaru-Colors is a fork of upstream Yaru, and it's not something that should be used anymore.

**Comment by 3v1n0** [comment]
So something like this (untested) should be done: https://github.com/3v1n0/org.gtk.Gtk3theme.Yaru-Red-dark/commit/a11a8b9b6f0e27864b84079961f46f137c5308fa

**Comment by 3v1n0** [comment]
As mentioned by @ernstp it's very likely that the name case mismatch is causing issues though, so in such case we should probably open a new repo for this?

**Comment by Mek101** [comment]
The flatpak's name is determinated by the [id](https://github.com/flathub/org.gtk.Gtk3theme.Yaru-Red-dark/blob/b1e8b8e2e56e503743c877c2e91c8829056b7790/org.gtk.Gtk3theme.Yaru-Red-dark.json#L2) field, so we should be able to just change the capitalization and it should be fine

**Comment by Mek101** [comment]
The yaru colors are bark, sage, olive, viridian, prussiangreen, blue, purple, magenta and red right?
https://github.com/ubuntu/yaru/blob/master/common/accent-colors.scss.in

**Comment by Mek101** [comment]
Although this would break compatibility with Focal Fossa users still using Yaru-Colors

**Comment by Mek101** [comment]
> The flatpak's name is determinated by the [id](https://github.com/flathub/org.gtk.Gtk3theme.Yaru-Red-dark/blob/b1e8b8e2e56e503743c877c2e91c8829056b7790/org.gtk.Gtk3theme.Yaru-Red-dark.json#L2) field, so we should be able to just change the capitalization and it should be fine

Although this would break compatibility with Focal Fossa users still using Yaru-Colors

**Comment by santiagofn** [comment]
I've tried other colors (like _Yaru-olive-dark_) and the same thing happens there: differente case and lack of support for accent colors.
Seems like https://github.com/flathub/org.gtk.Gtk3theme.Yaru-Red-dark/issues/1#issuecomment-1089184601 should be applied to all the variants.

**Comment by todoneunl** [comment]
Any update? Purple Dark is my favorite theme, but some apps are displayed as the light default. The Prussian Green Dark is (for example) displayed like it should on these apps.

**Comment by amardhruva** [comment]
Purple Light User here. I would love to get my flatpak apps themed.
Btw is there something blocking from creating two repos one with legacy support and one with lowercase new themes?
